hogvm/typescript/src/__tests__/bytecode.test.ts

```diff
@@ -0,0 +1,99 @@
+import { executeHogQLBytecode, Operation as op } from '../bytecode'
+
+describe('HogQL Bytecode', () => {
+    test('execution results', () => {
+        const fields = { properties: { foo: 'bar' } }
+        expect(executeHogQLBytecode(['_h', op.INTEGER, 2, op.INTEGER, 1, op.PLUS], fields)).toBe(3)
```

Comment:
It might be nice to re-use the generated bytecode from the python tests e.g. by storying them in a JSON file keyed by HogQL expression.